java/se/sics/mspsim/core/Timer.java

```diff
@@ -961,7 +961,7 @@ public void interruptServiced(int vector) {
     /* triggerInterrupts(cpu.cycles); */
   }
 
-  public int getModeMax() {
+  public static int getModeMax() {
```

Comment:
I think this method is a leftover from an earlier API and should be removed as it is now unused.